tests/aws/services/ec2/test_ec2.py

```diff
@@ -482,6 +482,87 @@ def test_create_vpc_with_custom_id(self, aws_client, create_vpc):
         assert e.value.response["ResponseMetadata"]["HTTPStatusCode"] == 400
         assert e.value.response["Error"]["Code"] == "InvalidVpc.DuplicateCustomId"
 
+    @markers.aws.only_localstack
```

Comment:
Ideally we want to verify both tests against AWS with snapshots. Take a look [here](https://blog.localstack.cloud/2022-08-04-parity-explained/) for more details. This is both for responses and exceptions. You can find several examples in other tests.